BatteryPackage.Darwin.csproj

```diff
@@ -0,0 +1,16 @@
+<Project Sdk="Microsoft.NET.Sdk">
+    <PropertyGroup>
+        <TargetFramework>net9.0</TargetFramework>
+        <PackageId>IronRe2-Batteries.OSX</PackageId>
+    </PropertyGroup>
+
+    <ItemGroup>
+
+    <Content Include="bin\cre2\libcre2.dylib">
+      <PackagePath>runtimes\osx\native\libcre2.dylib</PackagePath>
```

Comment:
Can we include the full rid here. `osx-x64` or `osx-arm64`. Ideally we'd build _both_ RIDs for this package.

Reply:
Added